core/message-tags-3.3.md

```diff
@@ -135,9 +137,14 @@ Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client
 ## Security considerations
 
 Client-only tags should be treated as untrusted data. They can contain any value
-and are not validated by servers in any way.
+and are not validated by servers in any way. The server MAY normalise values by applying the unescape
```

Comment:
This is intentional, I can't submit a PR on behaviour (#315) that doesn't exist.